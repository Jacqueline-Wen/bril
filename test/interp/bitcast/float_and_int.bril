@main() {
  as_int: int = const 4591870180066957722;
  as_float: float = bitcast as_int;
  print as_int as_float;
}
