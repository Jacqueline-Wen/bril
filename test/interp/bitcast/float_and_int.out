4591870180066957722 0.10000000000000001
